docs/release-notes/Language.md

```diff
@@ -0,0 +1,21 @@
+﻿---
+category: Release Notes
+categoryindex: 600
+index: 2
+title: F# Language
+---
+
+# F\# Language
+
+## Unreleased
+
+### Added
+
+* Better generic unmanaged structs handling. ([Language suggestion #692](https://github.com/fsharp/fslang-suggestions/issues/692), [PR #12154](https://github.com/dotnet/fsharp/pull/12154))
+* Bidirectional F#/C# interop for 'unmanaged' constraint. ([PR #12154](https://github.com/dotnet/fsharp/pull/12154))
+
+## 8.0.0 - 2023-11-14
```

Comment:
We have only 2 digits for the language version, 3-digit ones are invalid values.